docs/visualization/latency/index.md

```diff
@@ -46,7 +45,6 @@ The vertical axis means execution time of callback function, labeled as `Latency
 ## Communication
 
 `Plot.create_latency_timeseries_plot(communications: Collection[Communication])` is called when you are concerned how long time is consumed from message publish to corresponding subscription.
```

Comment:
It is better to insert the following sentences as well as other parts.

> Here, CARET takes into account communication when both transmission and reception on a message are performed successfully without being lost.
> See [Premise of communication](../premise_of_communication.md) for more details.

Reply:
Fixed at https://github.com/atsushi421/CARET_doc/pull/1/commits/ff485321764509954a3b9910c772718427a01976